muffnn/mlp/mlp_classifier.py

```diff
@@ -53,6 +53,13 @@ class MLPClassifier(MLPBaseEstimator, ClassifierMixin):
         If int, the random number generator seed. If RandomState instance,
         the random number generator itself. If None, then `np.random` will be
         used.
+    solver: an instance of a subclass of `tf.train.Optimizer`, optional
```

Comment:
"instance"?  Doesn't it take the class itself, not an instance?  e.g., the default is `tf.train.AdamOptimizer`